contracts/account/Account.sol

```diff
@@ -20,36 +20,4 @@ import {AccountCore} from "./AccountCore.sol";
  * NOTE: To use this contract, the {ERC7739-_rawSignatureValidation} function must be
  * implemented using a specific signature verification algorithm. See {SignerECDSA}, {SignerP256} or {SignerRSA}.
  */
-abstract contract Account is AccountCore, ERC721Holder, ERC1155Holder, ERC7739 {
-    bytes32 private constant PACKED_USER_OPERATION =
-        keccak256(
-            "PackedUserOperation(address sender,uint256 nonce,bytes initCode,bytes callData,bytes32 accountGasLimits,uint256 preVerificationGas,bytes32 gasFees,bytes paymasterAndData)"
-        );
-
-    /**
-     * @dev Specialization of {AccountCore-_signableUserOpHash} that returns a typehash following EIP-712 typed data
-     * hashing for readability. This assumes the underlying signature scheme implements `signTypedData`, which will be
-     * the case when combined with {SignerECDSA} or {SignerERC7702}.
-     */
-    function _signableUserOpHash(
-        PackedUserOperation calldata userOp,
-        bytes32 /*userOpHash*/
-    ) internal view virtual override returns (bytes32) {
-        return
-            _hashTypedDataV4(
-                keccak256(
-                    abi.encode(
-                        PACKED_USER_OPERATION,
-                        userOp.sender,
-                        userOp.nonce,
-                        keccak256(userOp.initCode),
-                        keccak256(userOp.callData),
-                        userOp.accountGasLimits,
-                        userOp.preVerificationGas,
-                        userOp.gasFees,
-                        keccak256(userOp.paymasterAndData)
-                    )
-                )
-            );
-    }
-}
+abstract contract Account is AccountCore, ERC721Holder, ERC1155Holder, ERC7739 {}
```

Comment:
I would consider removing `Account` whatsoever and just enable `ERC721Holder`, `ERC1155Holder`, `ERC721` and `ERC7739` as toggle features in Wizard.

We can keep this contract until we have the wizard version

Reply:
Note: this will require a lot of mocks changes, so I'd definitelly do it in a separate PR.